Orange/widgets/data/owfeatureconstructor.py

```diff
@@ -663,6 +652,68 @@ def freevars(exp, env):
         raise ValueError(exp)
 
 
+def validate_exp(exp):
+    """
+    Validate an `ast.AST` expression.
+
+    Only expressions with no list,set,dict,generator comprehensions
```

Comment:
I've been meaning to introduce a ListVariable type. Seems like I wouldn't be able to populate it with feature constructor?